src/index.html

```diff
@@ -4,9 +4,358 @@
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__content">
+        <a href="#" class="logo">
+          <img src="./images/logo.png" alt="moyo logo">
+        </a>
+
+        <nav>
+          <ul class="nav">
+            <li class="nav__item">
+              <a href="/" class="nav__link is-active">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                  Laptops & Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <article class="product-card" data-qa="card">
+          <div class="product-card__img-wrapp">
+            <img
+              class="product-card__img"
+              alt="Apple A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+              src="images/imac.jpeg">
+          </div>
+
+          <div class="product-card__headline">
+            <h3 class="product-card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+
+            <p class="product-card__subtitle">Product code: 195434</p>
+          </div>
+
+          <div class="product-card__review-wrapp">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="product-card__review-content">Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price-wrapp">
+            <p class="product-card__price-txt">Price:</p>
+
+            <p class="product-card__price">$2,199</p>
+          </div>
+
+          <a
+            class="product-card__cta"
+            href="#"
+            data-qa="card-hover">
+              Buy
+          </a>
```

Comment:
Fix formatting in all places
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/1c111a03-0d2a-402e-97c0-cb7565a382cc)